frontend/src/components/OperationsTableCard/OperationsTableCard.tsx

```diff
@@ -23,6 +24,9 @@ import { OperationsTableRow } from '../OperationsTableRow';
 import OperationsTableRowActions from '../OperationsTableRowActions';
 import { PaginationRow } from '../PaginationRow';
 import { SearchInput } from '../SearchInput';
+import { toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import { ExportToCsv } from 'export-to-csv';
```

Comment:
This library hasn't been updated in several years so it won't do ... let's keep things in papa parse https://stackoverflow.com/questions/52240221/download-save-csv-file-papaparse

Reply:
Removed old library and now using native `js` api